<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellingView</class>
 <widget class="QWidget" name="SellingView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Stock:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Filter:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="butClear">
           <property name="text">
            <string>clr</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filter"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="view">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="animated">
          <bool>false</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Code</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Count</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Discont</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Final Price</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>&amp;Basket:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Bonus card:</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="butSelectCard">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>none</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Bonuses:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cardValue">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Owner:</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cardOwner">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="butClearCard">
           <property name="text">
            <string>clr</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="basket">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="headerMinimumSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>#</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Code</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Count</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Discount</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Final price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Bonuses</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Total positions:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalPos">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Total discount:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalDisount">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Total bonuses:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalBonuses">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Total price:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalPrice">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="process">
           <property name="text">
            <string>Process</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>view</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>SellingView</receiver>
   <slot>ItemToBasket(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>SellingView</receiver>
   <slot>onClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>796</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>process</sender>
   <signal>clicked()</signal>
   <receiver>SellingView</receiver>
   <slot>onProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>886</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>butClear</sender>
   <signal>clicked()</signal>
   <receiver>filter</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filter</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SellingView</receiver>
   <slot>onChange(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>butClearCard</sender>
   <signal>clicked()</signal>
   <receiver>SellingView</receiver>
   <slot>onClearCard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>912</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>butSelectCard</sender>
   <signal>clicked()</signal>
   <receiver>SellingView</receiver>
   <slot>onSelCard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ItemToBasket(QTreeWidgetItem*,int)</slot>
  <slot>onClear()</slot>
  <slot>onProcess()</slot>
  <slot>onChange(QString)</slot>
  <slot>onClearCard()</slot>
  <slot>onSelCard()</slot>
 </slots>
</ui>
